vortex-file/src/strategy.rs

```diff
@@ -94,14 +96,26 @@ struct BtrBlocksCompressedStrategy {
 impl LayoutStrategy for BtrBlocksCompressedStrategy {
     fn new_writer(&self, ctx: &ArrayContext, dtype: &DType) -> VortexResult<Box<dyn LayoutWriter>> {
         let child = self.child.new_writer(ctx, dtype)?;
-        Ok(BtrBlocksCompressedWriter { child }.boxed())
+        Ok(BtrBlocksCompressedWriter {
+            child,
+            previous_chunk: None,
+        }
+        .boxed())
     }
 }
 
+struct PreviousCompression {
+    chunk: ArrayRef,
+    ratio: f64,
+}
+
+const COMPRESSION_DRIFT_THRESHOLD: f64 = 2.0;
```

Comment:
arbitrary value here

Reply:
It should probably be somewhat configurable, I'll do that in a followup.

Reply:
Interestingly previously we have chosen 1.2 as a value https://github.com/spiraldb/vortex/pull/996/files#diff-1e68a72976394481367e1fbac1fc8eb6706fd9123def2c7b1a1e0ce09dddcbd7R21